### PR TITLE
[1809] Withdraw course updates

### PR DESCRIPTION
**Pull request by danielburnley**
### Context

The withdraw course check had it's if condition backwards, this needed to be flipped.

Additionally, we needed to update the `content_status` to `withdrawn` so the frontend could respond appropriately

### Changes proposed in this pull request
- Flip the check
- Edit the content status

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x]  Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

File: spec/models/course/withdraw_spec.rb [hunk 1]
```diff
@@ -12,7 +12,9 @@
       course.withdraw
     end
 
-    context "an unpublished course" do
+    context "an published course" do
```
Comment (fofr) [inline spec/models/course/withdraw_spec.rb]: `an published`

**Comment by fofr** [comment]
Can you add a test for `course.is_published?` for the withdrawn content_status.

Is a withdrawn course published?
 - We still need to sync it so we update the sites on Find correctly
 - We probably still want to disable editing of some fields

Does that mean `is_published?` should be renamed?

Would that method be better updated to check for a last published date? 🤔

File: spec/models/course/withdraw_spec.rb [hunk 2]
```diff
@@ -12,9 +12,19 @@
...
+    context "a published course" do
+      let(:enrichment) { build(:course_enrichment, :published) }
+
       it "should not be findable" do
-        expect(course.findable?).to be_falsey
+        expect(course.findable?).to eq(false)
+      end
+
+      it "should not be published" do
+        expect(course.is_published?).to eq(false)
```
Comment (fofr) [inline spec/models/course/withdraw_spec.rb]: This test says:
"a published course … should not be published"

**Comment by danielburnley** [comment]
When a course is withdrawn it'll be deleted in the next bulk sync - as it's no longer marked as `published` or `syncable` :tada:

**Comment by PeteWilliams** [comment]
[Update prototype to reflect current live journey rejection reasons journeys](https://trello.com/c/1PEHPJlH/2257-update-prototype-to-reflect-current-live-journey-rejection-reasons-journeys)

**Pull request merged**